lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -10,9 +10,13 @@ const ReactDom = require('react-dom');
 const Modal = require('dosomething-modal');
 
 function getLocalStorageKey(user, campaign) {
+  // @TODO: probably need a better way to get the run ID consistently.
+  const languageCode = (Object.keys(campaign.campaign_runs.current)[0]);
+
   let key = null;
+
   if (campaign && user) {
-    key = `onboarding:campaign=${campaign.id}-user=${user.info.id}`;
+    key = `onboarding:campaign=${campaign.id}&run=${campaign.campaign_runs.current[languageCode].id}${user.info ? "&user=" + user.info.id :  ''}`;
```

Comment:
Feels weird to me having two types of keys looking at it, should it just be one or the other? or even something as simple as this?

`campaign=${campaign.id}&user=${user.info ? user.info.id :  -1}`